java/src/main/resources/MongoDBSample.java

```diff
@@ -0,0 +1,29 @@
+package sh.platform.languages;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+import sh.platform.config.Config;
+import sh.platform.config.MongoDB;
+
+import static com.mongodb.client.model.Filters.eq;
+
+public class MongoDBSample {
+
+    public static void main(String[] args)  {
```

Comment:
All of these examples should be spaced out better and commented.  Remember, they will show up in the docs, so assume they'll be read by customers 90x more often than executed.  They're documentation that happens to be executable.